submissions/cynthiaNwakaeme/HTML/index.html

```diff
@@ -0,0 +1,172 @@
+                        of organic ingredients, herbs and refined essential oil. 
+    
+                        <br>
+                        <br>
+    
+                        <b>Benefits of J-Royalty Organic African black soap</b>
+                        <ul>
+                            <li>It's antibacterial</li>
+                            <li>It's safe for all skin types</li>
+                            <li>It's moisturizing</li>
+                            <li>It helps soothe irritation</li>
+                            <li>It's anti-inflammatory</li>
+                            <li>It helps fight acne</li>
+                            <li>It helps reduce fine lines and improves skin texture</li>
+                            <li>It helps protect against photoaging</li>
+                            <li>It helps minimize the appearance of scars and stretch marks by
+                                 exfoliating whilst rejuvenating the growth of new skin</li>
+                            <li>It's antifungal</li>
+                            <li>It helps reduce hyperpigmentation</li>
+                        </ul>
+                        <br>
+                        <em>Direction</em>
+                        Apply Neon soap to the entire body, gently massage and 
+                        enjoy the pure delight of our essential oil bath sensation.
+                    </p>
+                </div>
+        </main>
+        <aside>
+            <div>
+                <h4>Shipping</h4>
+                <p>
+                    Payment validates orderNormally delivered between 5 to  10 working days after payment.
+                    Please check exact dates in the Order page.
+                    </p>
+            </div>
+            <hr>
+            <div>
+                <h4>Return Policy</h4>
+                <p>Free return within 3 days for Taries Empire items and 2 days for other eligible items.</p>
+            </div>
+            <hr>
+            <div>
+                <h4>Review</h4>
+                <span><h5>Andy Adams &#9733;&#9733;&#9733;&#9733;&#9733;</h5>
+                    <p>This Soap worked wonders. After 2 weeks use my acne faded off </p>
+                </span>
+                <span><h5>Bob Risky &#9733;&#9733;&#9733;&#9734;&#9734;</h5>
+                    <p>It worked but it is itch on my skin</p>
+                </span>
+                <span><h5>Susan Patrick &#9733;&#9733;&#9733;&#9733;&#9734;</h5>
+                    <p>Top notch delievery and customer servies. This soap fregrance is everyhing</p>
+                </span>
+                <em><a href="#">More reviews..</a></em>
+            </div>
+        </aside>
+    </section>
+    <section>
+        <hr>
+        <h2>You May like</h2>
+        <ul>
+            <oi>
+                <img src="img/black soap 2.jpg" alt="" width="10%">
+                <h4>Organic African Soap</h4>
+            </oi>
```

Comment:
<oi> is an invalid html element. i think you are trying to write <ol>

Reply:
I have effected the changes. kindly review